inst/template/default.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang = {{ lang }}>
```

Comment:
What happens if `lang` is `NULL`?

Reply:
When it's NULL, lang is set to "". This means assisitve technologies and search parsers will use browsers' default language setting.

Reply:
I think this needs quotes, as in:

```
<html lang = "{{ lang }}">
```